seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3458,6 +3456,9 @@
         "assert(code.match(/break/g).length > 2, 'message: You should have at least 3 <code>break</code> statements');"
       ],
       "type": "waypoint",
+      "MDNlinks": [
```

Comment:
@raisedadead I had edited with COM1000 for some reason that line was moved when it was saved.